docs/community/contribute/contribute_docs.md

```diff
@@ -0,0 +1,55 @@
+(contribute_docs)=
+
+# Contributing documentation
+
+```{eval-rst}
+.. tags:: Contribute, Basic
+```
+
+Whether you're a novice or experienced software engineer, data scientist, or machine learning
+practitioner, we welcome your contributions to the Flyte documentation!
+
+The Flyte documentation comprises the following types:
+
+* **{ref}`User guide <userguide>` documentation:** Conceptual and procedural documentation about using Flyte features to accomplish tasks.
+* **{ref}`Tutorials <tutorials>`:** Longer, more advanced guides that use multiple Flyte features to solve real-world problems. Some tutorials may require extra setup, while others can only run on larger clusters.
+* **{ref}`Integrations examples <integrations>`:** These examples showcase how to use the Flyte plugins that integrate with the broader data and machine learning ecosystem.
+* **{ref}`Deployment documentation <deployment>`:** Guidance on deploying and configuring the Flyte backend.
+* **{doc}`API documentation <../../api/index>`:** flytekit, flytectl, and flyteidl documentation.
+
+For minor edits that don't require a local setup, you can edit the page in GitHub page to propose improvements.
+
+## Contributing to user guide and deployment documentation
+
+To update user guide or deployment documentation, edit the corresponding files in the [flyte repository](https://github.com/flyteorg/flyte/tree/master/docs/user_guide).
+
+### Code in user guide documentation
+
+If you want to include tested, runnable example code in user guide documentation, you will need to add your code to the examples directory of the [flytesnacks repository](https://github.com/flyteorg/flytesnacks). Write your code in regular Python, with regular comments. These comments **will not** be extracted from the Python file and turned into user-facing documentation. You can use the `rli` ([remoteliteralinclude](https://github.com/wpilibsuite/sphinxext-remoteliteralinclude/blob/main/README.md)) directive to include snippets of code from your example Python file.
```

Comment:
This sentence reads oddly: "These comments **will not** be extracted from the Python file and turned into use-facing documentation"

The reader may be left wondering why exactly this particular fact about the comments is being called out.

Reply:
Good point -- this is to differentiate user guide examples in flytesnacks from tutorials and integrations examples in that repo, which have their comments turned into rendered docs. I can make that more explicit.